src/tools/dump-apiset/dump-apiset.cpp

```diff
@@ -0,0 +1,118 @@
+#include <cstdio>
+#include "platform/platform.hpp"
+#include "utils/compression.hpp"
+#include "utils/io.hpp"
+#include <vector>
+#include <iostream>
+
+void print_apiset(PAPI_SET_NAMESPACE apiSetMap);
+void create_header_file(const std::vector<uint8_t>& data);
+
+__forceinline PVOID GetCurrentProcessPeb()
+{
+#ifdef _WIN64
+    return (PVOID)__readgsqword(0x60);
+#else
+    return (PVOID)__readfsdword(0x30);
+#endif
+}
+
+int main()
+{
+    printf("Dump API-SET\n");
+    printf("------------\n\n");
+
+    const auto peb = (PPEB64)GetCurrentProcessPeb();
```

Comment:
do we have PEB32 structs already? might make sense to just use PPEB here so that it defaults to PEB32 on x86 processes, as GetCurrentProcessPeb supports 32 bit